packages/components/ovh-shell/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@ovh-ux/shell",
+  "version": "0.0.0",
+  "description": "Communication and interaction between applications",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/components/ovh-shell"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "sideEffects": "false",
+  "type": "module",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc -w"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "^5.0.0 || ^6.0.0",
```

Comment:
**issue(blocking):** Can't use private package.

Reply:
package "ovh-shell" has been made private for now, we'll look into it later if there is a need in making this package private, thanks